config/locales/javascript/javascript.en.yml

```diff
@@ -246,6 +246,7 @@ en:
     notifications:
       mark_read: "Mark read"
       mark_unread: "Mark unread"
+      new_notifications: "You have new notifications"
```

Comment:
Hmm... There is a problem using "count" as a variable name for localization. I suppose probably because of [this line](https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/helpers/i18n.js#L57). I throws `Error: Missing translation: notifications.new_notifications.other  i18n.js:48:15`. I used `number` instead.